### ISSUE TITLE
Node.js 12 actions are deprecated

### ISSUE THREAD

**Issue by guettli** [pascalgn/size-label-action#35]
I get this error in Github Action:

```
Node.js 12 actions are deprecated. Please update the following actions to use Node.js 16

pascalgn/size-label-action@1619680c5ac1ef360b944bb56a57587ba4aa2af8. 

```

For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/

Dear project maintainer, is there anything I can do to help?

**Comment by guibranco** [comment]
@pascalgn could you please view this and the related PR and approve it ?

**Comment by pascalgn** [comment]
Apologies for the long delay and thanks for raising this issue! It's fixed now and released as `pascalgn/size-label-action@v0.5.0`
